index.js

```diff
@@ -2,35 +2,48 @@
 
 const connoisseur = require("./lib/connoisseur");
 const fs = require("fs");
+const concat = require("concat-stream");
 const sourceFiles = require("yargs").argv._;
 
+const useStdIn = process.argv.indexOf("--stdin") > -1;
```

Comment:
This shouldn't be necessary. Unix programs accept input from `stdin` by default, without any parameters. If we need a flag, it should be in the other direction, when we accept a file name or a path.